pythoness/pythoness_module.py

```diff
@@ -122,16 +125,18 @@ def close(self):
         self.connection.close()
 
 
-def complete(user_prompt):
+def complete(user_prompt,history):
```

Comment:
Is `history` a placeholder for future chat interactions? Also, please add types and docstrings.

Reply:
Pls also add space after comma.

Reply:
Yes, this is a placeholder for keeping track of the conversation which will be used in the future. Added the types and docstrings and updating the pull request now.